packages/chart/components/LineChart.tsx

```diff
@@ -0,0 +1,39 @@
+`;
+export interface LineChartProps {
+  data: {
+    [label: string]: number;
+  };
+  type: string;
+}
+
+class LineChart extends React.PureComponent<LineChartProps, {}> {
+  public static defaultProps = {
+    type: "Line"
```

Comment:
Why can you change LineChart to a different chart?

Reply:
Good catch. Forgot to remove thins when I changed this to LineChart.